### PR TITLE
expression: implement vectorized evaluation for 'builtinUUIDSig'

### PR DESCRIPTION
**Pull request by tsthght**
### What problem does this PR solve? 
expression: implement vectorized evaluation for 'builtinUUIDSig', for #12106

### What is changed and how it works?
```
BenchmarkVectorizedBuiltinMiscellaneousCasesFunc/builtinUUIDSig-VecBuiltinFunc-8         	    5000	    276740 ns/op	   49152 B/op	    1024 allocs/op
BenchmarkVectorizedBuiltinMiscellaneousCasesFunc/builtinUUIDSig-NonVecBuiltinFunc-8      	    5000	    298279 ns/op	   49152 B/op	    1024 allocs/op
```

### Check List
Returns random numbers, so cannot write test cases

### CODE REVIEW

**Comment by SunRunAway** [comment]
@Reminiscent @qw4990 PTAL at these sort of functions, we need the benchmark.

File: expression/builtin_miscellaneous_vec.go [hunk 1]
```diff
@@ -15,6 +15,7 @@ package expression
 
 import (
 	"encoding/binary"
+	"github.com/google/uuid"
```
Comment (SunRunAway) [inline expression/builtin_miscellaneous_vec.go]: Please group this line into github.com/...

Comment (tsthght) [reply expression/builtin_miscellaneous_vec.go]: ok, i will refine it

**Comment by qw4990** [comment]
> @Reminiscent @qw4990 PTAL at these sort of functions, we need the benchmark.

You can post the benchmark result for this function here @tshqin .
For correct testing, do you have any ideas? @XuHuaiyu @SunRunAway

**Comment by XuHuaiyu** [comment]
I think we can write an individual test for `UUID` and `RAND` or extend the exist `TestUUID` function. @qw4990

**Pull request merged**